<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_form</class>
 <widget class="QMainWindow" name="admin_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Databases</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_DelDB">
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/db/del</normaloff>:/db/del</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QListWidget" name="listWidget_DB"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_AddDB">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/db/add</normaloff>:/db/add</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_Questions">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/questions/question</normaloff>:/questions/question</iconset>
       </attribute>
       <attribute name="title">
        <string>Questions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0" colspan="3">
         <widget class="QTabWidget" name="tabWidget_Questions">
          <property name="minimumSize">
           <size>
            <width>465</width>
            <height>300</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_test">
           <attribute name="title">
            <string>Test</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QTreeWidget" name="treeWidget_test_questions">
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>is_theme</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>id</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_learn">
           <attribute name="title">
            <string>Learn</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0">
             <widget class="QTreeWidget" name="treeWidget_learn_questions">
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>is_theme</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>id</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_Edit_Quest">
          <property name="maximumSize">
           <size>
            <width>157</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_Del_Quest">
          <property name="maximumSize">
           <size>
            <width>158</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="toolButton_Add_Quest">
          <property name="minimumSize">
           <size>
            <width>158</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tests</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="0">
           <widget class="QTreeWidget" name="treeWidget_tests">
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>is_theme</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>id</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_createTest">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_EditTest">
              <property name="maximumSize">
               <size>
                <width>157</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_DelTest">
              <property name="maximumSize">
               <size>
                <width>158</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Students">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/stud/group</normaloff>:/stud/group</iconset>
       </attribute>
       <attribute name="title">
        <string>Students</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_Delete_Stud">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="pushButton_Import_Stud">
            <property name="text">
             <string>Import</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/db/import</normaloff>:/db/import</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="6">
           <widget class="QTreeWidget" name="treeWidget_students">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>id</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QToolButton" name="toolButton_Add_Stud">
            <property name="minimumSize">
             <size>
              <width>104</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::MenuButtonPopup</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_Edit_Stud">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_EMail">
       <attribute name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/mail/email</normaloff>:/mail/email</iconset>
       </attribute>
       <attribute name="title">
        <string>EMail</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_SendEMail">
            <property name="title">
             <string>Send results of testing by email</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_email">
               <item row="1" column="0" colspan="2">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Recipients:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QTableWidget" name="tableWidget_email">
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="columnCount">
                  <number>3</number>
                 </property>
                 <column>
                  <property name="text">
                   <string>Recipient name</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Address</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>id</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChange_admin_password"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChange_admin_password">
   <property name="icon">
    <iconset>
     <normaloff>:/resourse/settings</normaloff>:/resourse/settings</iconset>
   </property>
   <property name="text">
    <string>Change admin password</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
